lib/_http_server.js

```diff
@@ -533,8 +537,12 @@ function socketOnClose(socket, state) {
 function abortIncoming(incoming) {
   while (incoming.length) {
     const req = incoming.shift();
+    // TODO(ronag): req.destroy(err)
     req.aborted = true;
     req.emit('aborted');
+    if (req.listenerCount('error') > 0) {
+      req.emit('error', connResetException('socket hang up'));
```

Comment:
Are we ok with using "socket hang up" on the server? It never existed before...

Reply:
I'm not all that concerned with the message, more the code. I think we used `aborted` as a message in the [other PR](https://github.com/nodejs/node/pull/28677). Would you prefer that?

Reply:
Yes, how about creating a new and dedicated `ERR_HTTP_ABORTED` error or something like that?

I don't have a desire to make `http` streams 100% compliant with other streams so I'm neutral here. As long as the `'error'` or the `'aborted'` listeners are not mandatory I'm with it.

Reply:
> Yes, how about creating a new and dedicated ERR_HTTP_ABORTED error or something like that?

I would prefer the error type to be consistent with `OutgoingMessage`. I'll change the message.

Reply:
changed to `'aborted'`

Reply:
While it's true that the `ECONNRESET` is used also for the "socket hang up" error (probably for compatibility reasons), I think it is misleading as there is no guarantee that the socket was closed with that error.

Reply:
So I guess it's a tradeoff between "correctness" and "consistency". For once I'm not on the side of "correctness". However, if you have a strong opinion on it I would not be opposed to changing it to what you propose.

Reply:
@mcollina: Do you have any thoughts on this matter?

Reply:
I would also say that it is also not consistent as it's a new and different error but I don't care that much.